docker/docker-compose-production.yaml

```diff
@@ -4,37 +4,56 @@ services:
     image: arkid15r/owasp-nest-backend:production
     env_file: .env.backend
     depends_on:
+      production-nest-cache:
+        condition: service_healthy
       production-nest-db:
         condition: service_healthy
     restart: unless-stopped
     networks:
       - nest-app-network
+      - nest-cache-network
       - nest-db-network
     volumes:
       - ./data:/home/owasp/data
 
-  production-nest-frontend:
-    container_name: production-nest-frontend
-    image: arkid15r/owasp-nest-frontend:production
+  production-nest-cache:
+    container_name: production-nest-cache
+    image: redis:7.2.7-alpine3.21
+    command: redis-server --requirepass nest_redis_production --maxmemory 25mb --maxmemory-policy allkeys-lru
```

Comment:
It seems this configuration uses plain text password to run the server. There is no point in a password if it's a known piece of information.